src/models/entities/library_items/disks/dvd/dvd.py

```diff
@@ -0,0 +1,10 @@
+from typing import Optional
+
+from pydantic import Field
+
+from src.models.entities.library_items.disks.disks import Disk
+
+
+class DVD(Disk):
+    director: str = Field(..., max_length=100, description="Director of the DVD")
```

Comment:
Why are these basemodel types? The point of an odm is to map the types automatically.